src/flow/darcy_flow_mh_output.hh

```diff
@@ -108,7 +108,7 @@ public:
     /** \brief Calculate values for output.  **/
     void output();
 
-    //const OutputFields &get_output_fields() { return output_fields; }
+    const OutputFields &get_output_fields() { return output_fields; }
```

Comment:
HM should not depend on the MH output. Why is this necessary?

Reply:
OK, removed in branch JS_coupling.